src/core/caniuse.js

```diff
@@ -0,0 +1,238 @@
+  cache.set(url, { stats, cacheTime: new Date() }).catch(console.error);
+  return stats;
+}
+
+/**
+ * get HTML element for the canIUse support table
+ * @param  {Object} stats     CanIUse API results
+ * @param  {Object} conf      respecConfig.caniuse
+ */
+function createTableHTML(conf, stats) {
+  let browsers = conf.browsers;
+  if (conf.browsers === "ALL") {
+    browsers = Object.keys(stats);
+  }
+
+  // render the support table
+  return hyperHTML`
+    ${browsers.map(browser =>
+      addBrowser(browser, conf.versions, stats[browser])
+    )}
+    <a href="${`http://caniuse.com/#feat=${conf.feature}`}"
+      title="Get details at caniuse.com">More info
+    </a>`;
+
+  /**
+   * add a browser and it's support to table
+   * @param {String} browser      name of browser (as in CanIUse API response)
+   * @param {Number} numVersions  number of old browser versions to show
+   * @param {Object} browserData  stats data from api response
+   */
+  function addBrowser(browser, numVersions, browserData) {
+    if (!browserData) return "";
+    const getSupport = version => {
+      const supportKeys = browserData[version]
+        .split("#", 1) // don't care about footnotes.
+        .reduce((collector, items) => collector.concat(items.split(" ")), [])
+        .filter(item => item);
+      let titles = [];
+      for (const key of supportKeys) {
+        // Keys from https://github.com/Fyrd/caniuse/blob/master/CONTRIBUTING.md
+        switch (key) {
+          case "y":
+            titles.push("Supported.");
+            break;
+          case "a":
+            titles.push("Almost supported (aka Partial support).");
+            break;
+          case "n":
+            titles.push("No support, or disabled by default.");
+            break;
+          case "p":
+            titles.push("No support, but has Polyfill.");
+            break;
+          case "u":
+            titles.push("Support unknown.");
+            break;
+          case "x":
+            titles.push("Requires prefix to work.");
+            break;
+          case "d":
+            titles.push("Disabled by default (needs to enabled).");
+            break;
+        }
+      }
+      return {
+        support: supportKeys.join(" "),
+        title: titles.join(" "),
+      };
+    };
+    const addBrowserVersion = version => {
+      const { support, title } = getSupport(version);
+      const cssClass = "caniuse-cell " + support;
+      return `<li class="${cssClass}" title="${title}">${version}</li>`;
+    };
+
+    const [latestVersion, ...olderVersions] = Object.keys(browserData)
+      .sort(semverCompare)
+      .slice(-numVersions)
+      .reverse();
+
+    const { support, title } = getSupport(latestVersion);
+    return hyperHTML`
+      <ul class="caniuse-browser">
+        <li class="${"caniuse-cell " + support}" title="${title}">
+          ${BROWSERS.get(browser) || browser} ${latestVersion}
+          <ul class="caniuse-col">
```

Comment:
This throws off the design a little bit, but this is the markup we want here. Otherwise, the lists are disjoint. We can probably make this a bit better... we should really only have one list per browser.

Reply:
I tried to do that at first. But I couldn't figure out CSS based hover (whatever I figured out was causing layout to move  :cry: ). Will try that again after some time.